worker/src/Channel/Request.cpp

```diff
@@ -57,9 +57,22 @@ namespace Channel
 	};
 	// clang-format on
 
+	/* Class variables. */
+
+	Channel::UnixStreamSocket* Request::channel{ nullptr };
```

Comment:
As told before, I need to understand why we need to make `Request::channel` static for this PR to work.

Reply:
(let's discuss this topic on the comment above)